custom_components/audiconnect/audi_services.py

```diff
@@ -577,6 +577,81 @@ async def set_climatisation(self, vin: str, start: bool):
             "action.actionState",
         )
 
+    async def start_climate_control(
+        self,
+        vin: str,
+        temp_f: int,
+        temp_c: int,
+        glass_heating: bool,
+        seat_fl: bool,
+        seat_fr: bool,
+        seat_rl: bool,
+        seat_rr: bool,
+    ):
+        target_temperature = None
+        if temp_f is not None:
+            target_temperature = int(((temp_f - 32) * (5 / 9)) * 10 + 2731)
+        elif temp_c is not None:
+            target_temperature = int(temp_c * 10 + 2731)
+
+        # Default Temp
+        target_temperature = target_temperature or 2941
+
+        # Construct Zone Settings
+        zone_settings = [
+            {"value": {"isEnabled": seat_fl, "position": "frontLeft"}},
```

Comment:
With these not being mandatory or defaults set wouldn't this pass none or null to the api?

Reply:
Defaults are defined in `[audi_account.py](https://github.com/audiconnect/audi_connect_ha/pull/315/files/fecd5af073b6f07eb38d4515c562a792c2354476#diff-c3f638333dff9dd695ea6ad54cb0a4f1b07b71f93c533a62de37b73fc9a0f8ab#L230-234)`

Reply:
Correct.  Thanks for pointing me to that!

Reply:
No worries! In retrospect it probably makes more sense to put it in the `audi_services.py` for the exact reason you didn't see it haha

But I'll note it as a to-do item when I have some time to test it out.